INFO     test_app:test_app.py:161 測試例外處理